modules/ocf_www/files/iptables-log.conf

```diff
@@ -0,0 +1,4 @@
+# Send iptables logs to a separate file
+# Must come before others to prevent other rules from recording first
+:msg,contains,"[iptables] " /var/log/iptables.log
```

Comment:
It's be nice if you could make this more specific since I could see other messages containing `[iptables] ` potentially, but that's not a big deal if some logs are sent to the wrong file. Looks like you could have a combination of this one plus another filter that it starts with `kernel` and that might work? That or maybe doing `[iptables] IN=` here to be more specific.

Maybe add some spaces in between commas to make it a bit more readable?

Might also need log rotation on this, not sure if that'll be done for you or not. Probably can wait and see on that one though.